tests/e2e.test.js

```diff
@@ -170,6 +159,23 @@ describe('Search', () => {
     },
   );
 
+  it('always show report buttons', async () => {
+    const $iconMissing = await page.$('#icon-missing');
+    const $iconOutdated = await page.$('#icon-outdated');
+    expect(await isVisible($iconMissing)).toBeTruthy();
+    expect(await isVisible($iconOutdated)).toBeTruthy();
+
+    const $searchInput = await page.$('#search-input');
+    await $searchInput.type(' ');
+    expect(await isVisible($iconMissing)).toBeTruthy();
+    expect(await isVisible($iconOutdated)).toBeTruthy();
+
+    await page.$eval('#search-input', (el) => (el.value = ''));
+    await $searchInput.type('litomore');
```

Comment:
Please, rewrite this as the old test, "litomore" is too much generic and doing fuzzy searching could lead to the test fail in the future. Better "this is definitely not going to match" or something like that.